content/en/docs/Contribution Guidelines/releasing.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: "Releasing Parquet"
-linkTitle: "Releasing Parquet"
+title: "Releasing Parquet-Java"
+linkTitle: "Releasing Parquet-Java"
 weight: 4
 description: >
-  How to release Parquet
+  How to release Parquet-Java
```

Comment:
Releasing parquet-format is pretty much the same process as releasing parquet-java. Perhaps we can add a line to explain this?

Reply:
Added below.